css/custom.css

```diff
@@ -949,7 +949,36 @@ canvas {
     }
 
 /* trusties */
-
+.trusties-section {
+    background-color:#F8F9FA;
+    padding-bottom: 48px;
+}
+.trusties-img {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 50px;
+}
+.trustie-avatar {
+    width: 160px;
+    height: 130px;
+}
+.trustie-avatar > img {
+    width: 160px;
+    height: 130px;
+}
```

Comment:
Tarpų gal norėtųsi tarp tų logotipų.. kokių 30pixeliukų (didžiausioje versijoje, responsive irgi reikia sužiūrėti).

Reply:
aha man irgi noretusi, bet uzdejus tarpus, kadangi musu bendras resposive nera tobulas, visa eilute iskart tada turetu nusokt i pavienus logotipus, nu tai is esmes jeigu neturi kompo kurio ekranas 1110px niekada nesimatytu eileje tie logotipai? 
Bet nezinau pabandyk pazasit per ispect gal atradai kiek paddingo arba margino padarytu tarpa neisokant is bendro container ribu

Reply:
Pakeiciau i didesnius tarpus,, tai pasiziurek patampyk ta ekrana

Reply:
oooo pas tave dabar viskas gerai? nes lyg atrodo daug kas nušoko.. (services ir education and experiance).. reik pažiūrėt kas ten kaip kodėl..